cli/src/commands/get.rs

```diff
@@ -353,14 +357,19 @@ pub fn run(get_args: &GetArgs, client: Client) -> Result<()> {
                 OutputFormat::Json => print_resources_as_json(users.iter(), io::stdout().lock())?,
             }
         }
-        GetArgs::CurrentUser { output } => {
+        GetArgs::CurrentUser { short, output } => {
             let user = client.get_current_user().chain_err(|| {
                 ErrorKind::Client("Operation to get the current user has failed.".into())
             })?;
-            match output {
-                OutputFormat::Table => print_users_table(&[user]),
-                OutputFormat::Json => {
-                    print_resources_as_json(iter::once(&user), io::stdout().lock())?
+
+            if *short {
+                println!("{}", user.username.0);
```

Comment:
I think `println!` panics if the write fails -- using `io:stdout()` + `write()` would return an error (when piping output this not a theoretical problem)